modin/pandas/test/utils.py

```diff
@@ -50,7 +56,7 @@
 random_state = np.random.RandomState(seed=42)
 
 DATASET_SIZE_DICT = {
-    "Small": (2**2, 2**3),
+    "Small": (2**6, 2**6),
```

Comment:
Right now it only affects this `BaseOnPython` execution. Due to the different number of columns, it is not very convenient to write tests, since this has to be taken into account. For now, it’s enough for me that the number of columns is no less than the number that is in normal mode.

Another option is to run all executions on the same data sizes, that is, use the normal dataset size.